### PR TITLE
Add types for express-ejs-layouts NPM package

### PR DESCRIPTION
**Pull request by erikmav**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not provide its own types, and you can not add them.
- [X] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

Note module is substantially empty (an Express request handler with low surface area), created manually.

### CODE REVIEW

File: types/express-ejs-layouts/express-ejs-layouts-tests.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import * as expressEjsLayouts from 'express-ejs-layouts';
+
+function expressRequestHandlerTest() {
+    const requestHandler = expressEjsLayouts();
+}
```
Comment (plantain-00) [inline types/express-ejs-layouts/express-ejs-layouts-tests.ts]: The test seems not right, you can use the code from the document as the test case, like:
```ts
import express = require('express');
import expressLayouts = require('express-ejs-layouts');

const app = express();
app.use(expressLayouts);
```

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/express-ejs-layouts/express-ejs-layouts-tests.ts (hunk 1 above)
Comment (erikma) [reply types/express-ejs-layouts/express-ejs-layouts-tests.ts]: Done

File: types/express-ejs-layouts/express-ejs-layouts-tests.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import * as express from 'express';
+import * as expressEjsLayouts from 'express-ejs-layouts';
+
+function expressRequestHandlerTest() {
+    const app = express()
+        .use(expressEjsLayouts());
+}
```
Comment (plantain-00) [inline types/express-ejs-layouts/express-ejs-layouts-tests.ts]: It needn't be put into a function, the test just checks, does not run.

**Review by plantain-00 — APPROVED** [review]

File: types/express-ejs-layouts/express-ejs-layouts-tests.ts (hunk 2 above)
Comment (erikma) [reply types/express-ejs-layouts/express-ejs-layouts-tests.ts]: True. I prefer test case methods though - in other DefinitelyTyped submissions I had multiple test variations. Leaving the test methods in there help maintainers realize it's possible to do.

**Pull request merged**